frontend/test/metabase/scenarios/question/filter.cy.spec.js

```diff
@@ -743,4 +743,30 @@ describe("scenarios > question > filter", () => {
     });
     cy.findByText("wilma-muller");
   });
+
+  it.skip("column filters should work for metrics (metabase#15333)", () => {
+    visitQuestionAdhoc({
+      dataset_query: {
+        type: "query",
+        query: {
+          "source-table": PRODUCTS_ID,
+          aggregation: [["count"]],
+          breakout: [["field-id", PRODUCTS.CATEGORY]],
+        },
+        database: 1,
+      },
+      display: "table",
+    });
+
+    cy.get(".cellData")
+      .contains("Count")
+      .click();
+    cy.findByText("Filter by this column").click();
+    cy.findByPlaceholderText("Enter desired number").type("42");
```

Comment:
The old legacy style input should not exist.
It should use the "new" (since 0.28 or something like that) style, just like the sidebar/Notebook
```suggestion
    cy.findByPlaceholderText("Enter a number").type("42");
```